cardinal/go.mod

```diff
@@ -11,6 +11,7 @@ replace (
 
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 	github.com/syndtr/goleveldb => github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
+	pkg.world.dev/world-engine/assert => ../assert
```

Comment:
i dont think we should do this replace here

Reply:
why.

Reply:
id prefer to use tagged versions (or a specific commit at worst case). we don't use replace for the sign or evm modules

Reply:
hm. Why is that way better? Wouldn't it be more convenient to always just use the local packages? That way unit tests and updates always are fresh and up to date? 

Right now that package doesn't actually exist. So it has to be used like this for now. If I am to change it, it will be in another PR.

Reply:
> Wouldn't it be more convenient to always just use the local packages?

this is what go.work does

> Right now that package doesn't actually exist. So it has to be used like this for now. If I am to change it, it will be in another PR.

you can do it from this PR, doesn't have to be another - either grab it via commit hash, or push a new tag from `assert` package

Reply:
hm seems to work with tests.